wrappers/python/examples/llama_training.py

```diff
@@ -50,42 +53,42 @@
 
 
 parser.add_argument("--model-path", default=".model")
+parser.add_argument("--seq-len", type=int, default=1024)
 parser.add_argument("--seq-len-tile", type=int, default=1024)
 parser.add_argument("--batch-size", type=int, default=1)
 parser.add_argument("--minibatch-size", type=int, default=1)
 parser.add_argument("--minibatch-size-tile", type=int, default=1)
-parser.add_argument("--n-embd-tile", type=int, default=384)
-parser.add_argument("--n-inner-tile", type=int, default=1536)
-parser.add_argument("--n-head-tile", type=int, default=-1)
+parser.add_argument("--hidden-size-tile", type=int)
+parser.add_argument("--intermediate-size-tile", type=int)
+parser.add_argument("--n-head-tile", type=int)
 parser.add_argument("--torch-device", choices=["cpu", "cuda", "cuda:0",
         "cuda:1", "cuda:2", "cuda:3", "cuda:4"], default="cpu")
 parser.add_argument("--torch-dtype", choices=["fp32", "fp64", "bf16"],
```

Comment:
Add `tf32` precision. Right now, `nntile_dtype` is checked to enable TF32 in PyTorch, which is strange.